tests/phpunit/tests/blocks/updateIgnoredHookedBlocksPostMeta.php

```diff
@@ -0,0 +1,225 @@
+<?php
+/**
+ * Tests for update_ignored_hooked_blocks_postmeta
+ *
+ * @package WordPress
+ * @subpackage Blocks
+ *
+ * @since 6.6.0
+ *
+ * @group blocks
+ */
+class Tests_Blocks_UpdateIgnoredHookedBlocksPostMeta extends WP_UnitTestCase {
+
+	/**
+	 * Original markup.
+	 *
+	 * @var string
+	 */
+	protected static $original_markup;
```

Comment:
We're currently not using this. We should either use it (and remove the local vars from each test; though one of them seems to slightly differ from the others), or remove it.

Reply:
34526ff2505e1a757f34228215ef3e5f395956c7